config.md

```diff
@@ -452,6 +458,57 @@ For example, on Linux this would happen before the `pivot_root` operation is exe
 
 The definition of `createContainer` hooks is currently underspecified and hooks authors, should only expect from the runtime that the mount namespace and different mounts will be setup. Other operations such as cgroups and SELinux/AppArmor labels might not have been performed by the runtime.
 
+### <a name="configHooksSendSeccompFd" />SendSeccompFd Hooks
+
+The `sendSeccompFd` hooks MUST only be called if the seccomp policy contains `SCMP_ACT_NOTIFY`.
+
+The `sendSeccompFd` hooks MUST be called after the [`start`](runtime.md#start) operation is called and after the seccomp policy is installed but [before the user-specified program command is executed](runtime.md#lifecycle).
+The `sendSeccompFd` hooks MAY additionally be called while the container is in the [`running` state](runtime.md#runtimeState), for example during an `exec` operation.
+The goal of this hook is to pass the seccomp file descriptor to a seccomp agent.
+
+The `sendSeccompFd` hooks' path MUST resolve in the [runtime namespace](glossary.md#runtime-namespace).
+The `sendSeccompFd` hooks MUST be executed in the [runtime namespace](glossary.md#runtime-namespace).
+
+### <a name="seccompstate" />The Seccomp State
+
+The seccomp state is a data structure passed via stdin to the SendSeccompFd hooks.
+It includes the following properties:
+
+* **`ociVersion`** (string, REQUIRED) is version of the Open Container Initiative Runtime Specification with which the seccomp state complies.
+* **`phase`** (string, REQUIRED) is the phase at which the seccomp file descriptor is created.
+    The value MAY be one of:
+
+    * `start`: the seccomp filter is created following the [`start`](runtime.md#start) command
+    * `exec`: the seccomp filter is created following an `exec` command
```

Comment:
I just realized the OCI spec doesn't define an `exec` command. Given that, I'm not sure about including this field in the state anymore. 

Maybe we could use `State.Status` to differentiate between a call starting the container or a call when the container is already running?

Reply:
sounds good if it works. Have you checked what does `State.Status` contain when the seccomp hook is called from the startup phase?

Reply:
Yes, it's "creating", hence the hook / agent is able to differentiate the two cases based on it.

Reply:
Checked, it is State.Status=`creating` and State.Status=`running`. Looks good.

Reply:
I removed the "phase" field.